custom/icds/location_reassignment/models.py

```diff
@@ -63,14 +63,16 @@ def valid(self):
             return False
         valid = True
         for old_location in self.old_locations:
-            if (DEPRECATED_TO in old_location.metadata
```

Comment:
Curious, was this necessary or is it a stylistic change?

Reply:
i added these fields to location custom fields and when i added a new location, these were already present which caused this to be true.